javascript/core/objects/converting-to-and-from-json.md

```diff
@@ -0,0 +1,78 @@
+---
+author: EwaTrzemzalska
```

Comment:
This should be the username for your Enki account. 

If you haven't created one, we will automatically assign your Github or Google username (depending on how you logged in). You can verify your Enki username in the Account settings in the app.

Reply:
Related: https://github.com/enkidevs/curriculum/wiki/Insight-Documentation#author-m

Reply:
Oh, it's Enki user, not GitHub user? 🤦‍♀️ I didn't get it from the documentation.